app/src/highlighter/globals.d.ts

```diff
@@ -0,0 +1,253 @@
+/* tslint:disable */
```

Comment:
We were violating a bunch of rules that we had to violate. Our tslint rules are not designed for declaration files describing another external library.

I've thought about upstreaming this but it's a pretty special snowflake in that it's all duplicates and omissions in order to avoid touching anything DOM-related. I think we could upstream it but I'm not so sure of how useful it would be.

Reply:
> We were violating a bunch of rules that we had to violate.

Oh yeah, that wasn't a problem. I just used the first line as a discussion marker.

> I think we could upstream it but I'm not so sure of how useful it would be.

:thumbsup: happy to leave it for now